giraffe/src/components/SimpleTable/PaginationNav/Button/Button.scss

```diff
@@ -0,0 +1,423 @@
+
```

Comment:
Button.scss is HUGE. There's too many definitions and styles in there. Let's cut it down to only the ones we are using.

Reply:
Removed unnecessary CSS and cut it down by about half. Color and Size was removed because SimpleTable components are going to be static. Thus, it's not going to change beyond medium size and tertiary color.

Reply:
Loading and Disabled was removed for 2 reasons. 

1. SimpleTable does not pass in any props from SimpleTableGraph.tsx that would include ComponentStatus.Disabled or ComponentStatus.Loading

2. If for some reason it does use these statuses and it was missed, observations on the UI side shows that ComponentStatus.Disabled is often used when a faulty option or invalid input is given. ComponentStatus.Loading is often used after the action of pressing the button sends a request somewhere, and that wait time to process said request means that it is “loading”. Both situations should not happen for our uses of Button.

Reply:
Note: `yarn built` ran but did not update the `d.ts` file.